### PR TITLE
Itsadssd 31507 revise alert component

### PR DESCRIPTION
**Pull request by uqtgeor4**

### CODE REVIEW

**Review by Bcrunk — COMMENTED** [review]
Icon Recommendation. 

To better match the existing icons used for Information and warning alerts, I recommend adopting a similar style. 

Success: Check Circle 1
Error: remove-circle

image
image

**Review by Bcrunk — COMMENTED** [review]
There is a slight alignment issue with the icons, they seem to be displaced lower than the text 

Review against the mock-ups but they look misaligned when there is a single line of text

**Review by Bcrunk — COMMENTED** [review]
Further to the structural discussion already had. 

"Global Alerts" are not a function of the Alert, It is merely a Use case / preference for how to use the alerts. 

I don't think in this case we should use the uq-alert--dark as a class to represent a use case. 

Instead I would make the --dark modifier only change to the alternative colour. 
and create a new modifier --global 
(I actually think the naming is wrong, maybe something more accurate relevant to the modifier purpose --banner --container)

Also rather than needing a second div to act as a container you could utilise "justify-content: center;"

File: packages/storybook-html/src/components/alert/alert.docs.mdx [hunk 1]
```diff
@@ -0,0 +1,303 @@
...
+```
+
+### Warning
+
+<Canvas withSource="none">
+  <Story id="components-alert--alert-warning" />
+</Canvas>
+
+```html
+<div class="uq-alert uq-alert-warning" role="alert">
```
[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/storybook-html/src/components/alert/alert.docs.mdx:53]: Done

File: packages/storybook-html/src/components/alert/alert.docs.mdx [hunk 2]
```diff
@@ -0,0 +1,303 @@
...
+</div>
+```
+
+### Error
+<Canvas withSource="none">
+  <Story id="components-alert--alert-error" />
+</Canvas>
+
+```html
+<div class="uq-alert uq-alert-error" role="alert">
```
[the remark this answers is not in the document]
Comment (uqtgeor4) [reply packages/storybook-html/src/components/alert/alert.docs.mdx:66]: Done

File: packages/icon/src/scss/_build/_icons-emit.scss [hunk 3]
```diff
@@ -107,6 +107,9 @@
 @include icon('how-youll-learn--performance-presentation-graph');
 @include icon('how-youll-learn--seo-search-page');
 @include icon('how-youll-learn--users');
+@include icon('interface-essential--check-circle-1');
```
Comment (uqmchu) [inline packages/icon/src/scss/_build/_icons-emit.scss:110]: You shouldn't need to edit the build files if you follow the icon package maintenance process. Also, I believe the reason why these new icons look different is that Jen or Laura actually modified the stroke widths for Future Students so the same needs to be done for these as well. Using the proper build process will also give you updated documentation.

Comment (uqmchu) [reply packages/icon/src/scss/_build/_icons-emit.scss:110]: They've included their export process here: https://www.figma.com/file/vG3eumt1NqHAIe0AQjkuCX/Design-system-WIP?node-id=232%3A0

**Review by uqmchu — COMMENTED** [review]
I'm just leaving some notes about modifying the icon package here

**Review by uqnm — APPROVED** [review]

**Pull request merged**